SOURCES.md

```diff
@@ -31,9 +31,15 @@ Read messages from a Slack channel (requires a working [bot token](https://api.s
 
 #### SlackSource Configuration Parameters
 
-* `channel` - The Slack channel to read messages from.
+* `channel` - The Slack channel ID to read messages from. You will find it on Slack after clicking 'Get channel details' in the top left corner of the opened channel.
```

Comment:
May be we can change to **Open channel details** instead of **Get channel details**.  Because After right clicking on channel in left menu it displays as Open channel details. So wont be confused with that

Reply:
@Srihari1192 , hello, during work #737 I found out, it is channel name, not channel ID, so this discussion is outdated now. See my updated / simplified PR please